### DISCUSSION TITLE
How to add multiple tag IDs?

### DISCUSSION THREAD

**Discussion by bargzeek** [johannschopplich/nuxt-gtag#43] [Q&A]
Is there a way to add additional ids or accounts? like in google-gtag?

**Comment by johannschopplich** [comment]
Actually, since [v1.2.0](https://github.com/johannschopplich/nuxt-gtag/releases/tag/v1.2.0), yes!


If you want to send data to multiple destinations, you can add more than one Google tag ID to your Nuxt configuration in the `tags` module option.

The following example shows how to load a second Google tag that is connected to a Floodlight destination. To send data to Floodlight (tag ID `DC-ZZZZZZZZZZ`), add another config command after initializing the first Google tag (tag ID `GT-XXXXXXXXXX`):

```ts
// `nuxt.config.ts`
export default defineNuxtConfig({
  modules: ['nuxt-gtag'],

  gtag: {
    tags: [
      'GT-XXXXXXXXXX', // Google Ads and GA4
      'DC-ZZZZZZZZZZ' // Floodlight
    ]
  }
})
```

Or use the object syntax to initialize multiple tags with different configurations:

```ts
// `nuxt.config.ts`
export default defineNuxtConfig({
  modules: ['nuxt-gtag'],

  gtag: {
    tags: [
      {
        id: 'GT-XXXXXXXXXX',
        config: {
          page_title: 'GA4'
        }
      },
      {
        id: 'DC-ZZZZZZZZZZ',
        config: {
          page_title: 'Floodlight'
        }
      }
    ]
  }
})
```
